src/_P098_PWM_motor.ino

```diff
@@ -198,6 +202,12 @@ boolean Plugin_098(uint8_t function, struct EventStruct *event, String& string)
       addFormNumericBox(F("Encoder Timeout"), F("enc_timeout"), P098_ENC_TIMEOUT, 0, 1000);
       addUnit(F("ms"));
 
+      addFormNumericBox(F("Virtual speed"), F("virtual_speed"), P098_VIRTUAL_SPEED, 0, 2147483647);
```

Comment:
Isn't speed by definition a position-to-time factor? It is "virtual" to emphasise that it is an estimate and should not be confused with pwm.
However, I am open for other names. Any suggestions?

Reply:
Problem is that something does need to ramp up and slow down.
Therefore it is not really a 1-to-1 relation to a position. Only when you always move for the same duration and no difference in direction.

I think, that's more of the problem I'm having with this parameter.
It is more like a runtime instead of a position.

Reply:
How can we continue here?

Reply:
Any progress?

Reply:
I think with your most recent commit, we probably can merge it.
Will have 1 more look at it later this evening